test/test_manifest.json

```diff
@@ -6553,5 +6553,20 @@
       "link": true,
       "type": "eq",
       "annotations": true
-   }
+   },
+   { "id": "bug1724918",
+      "file": "pdfs/bug1724918.pdf",
+      "md5": "ab30269570c8ff2c9f8eb73fb376a081",
+      "rounds": 1,
+      "type": "eq",
+      "annotations": true
+   },
+   { "id": "xfa_bug1724918",
+      "file": "pdfs/xfa_bug1724918.pdf",
+      "md5": "f61579018fbdbb7b7dc5527735513019",
+      "enableXfa": true,
+      "link": true,
+      "rounds": 1,
+      "type": "eq"
+    }
```

Comment:
Do we actually need this new test-case, given that it seems to be well-covered by existing tests?